### PR TITLE
Qt teleop app for turtlebot

### PR DESCRIPTION
**Pull request by jihoonl**

### CODE REVIEW

**Comment by bit-pirate** [comment]
No icon? 재미없어.

**Comment by stonier** [comment]
Can we rename this? It should just be turtlebot/teleop. It actually has nothing to do with qt - any remote control interaction/concert can control it. In fact both android and qt remocons will handle this one.

**Comment by jihoonl** [comment]
renamed it. icon 나중에

**Comment by bit-pirate** [comment]
Please rebase and squash those 3 commits into one. Thx.

File: turtlebot_core_apps/package.xml [hunk 1]
```diff
@@ -38,6 +38,7 @@
     <rocon_app>rapps/xbox360_teleop/xbox360_teleop.rapp</rocon_app>
     <rocon_app>rapps/follower/follower.rapp</rocon_app>
     <rocon_app>rapps/ps3_teleop/ps3_teleop.rapp</rocon_app>
+    <rocon_app>rapps/qt_teleop/qt_teleop.rapp</rocon_app>
```
Comment (bit-pirate) [inline turtlebot_core_apps/package.xml]: here it is still called qt_teleop instead of teleop.

File: turtlebot_core_apps/rapps/teleop/teleop.rapp [hunk 2]
```diff
@@ -0,0 +1,5 @@
+display:          Teleop
+description:      Control the TurtleBot with QT remocon
```
Comment (bit-pirate) [inline turtlebot_core_apps/rapps/teleop/teleop.rapp]: Here as well.

**Comment by jihoonl** [comment]
It has rebased as you requested.

**Comment by bit-pirate** [comment]
Thx.

**Pull request merged**